src/FilterWrapper/components/FilterActionsPopover/FilterActionsPopover.tsx

```diff
@@ -1,102 +1,84 @@
-import * as React from "react";
 import cn from "classnames";
-import { OverflowVisibleContainer } from "../../../OverflowVisibleContainer";
+import * as React from "react";
 import { FilterAction } from "../../types";
 
 import "./FilterActionsPopover.scss";
 
-import {
-	IsntNeutralZoneMarker
-} from "../../../WindowClickListener";
+import { IsntNeutralZoneMarker } from "../../../WindowClickListener";
 
 interface FilterActionsPopoverProps {
-	filterActions: (FilterAction | any)[];
+	filterActions: Array<FilterAction | any>;
 	filterActionsCaption: string;
 }
 
-interface State {
-	isOpen: boolean;
-}
-
 const isFilterAction = (object: any): boolean => {
 	const filterAction = object as FilterAction;
-	return filterAction["key"] !== undefined
-		&& filterAction["onClick"] !== undefined
-		&& filterAction["name"] !== undefined;
-}
+	return (
+		filterAction.key !== undefined &&
+		filterAction.onClick !== undefined &&
+		filterAction.name !== undefined
+	);
+};
 
-export class FilterActionsPopover extends React.Component<
-	FilterActionsPopoverProps,
-	State
-	> {
-	public refElement = React.createRef<HTMLElement>();
+export const FilterActionsPopover: React.FC<FilterActionsPopoverProps> = ({
+	filterActions,
+	filterActionsCaption
+}) => {
+	const [isOpen, setIsOpen] = React.useState(false);
 
-	public state = {
-		isOpen: false
+	const handleClickFilter = () => {
+		setIsOpen(!isOpen);
 	};
 
-	public handleClickFilter = () => {
-		this.setState(state => ({ isOpen: !state.isOpen }));
+	const hideListPopover = () => {
+		setIsOpen(false);
 	};
 
-	public hideListPopover = () => {
-		this.setState(state => ({ ...state, isOpen: false }));
-	}
-
-	public render() {
-		const { filterActions, filterActionsCaption } = this.props;
+	return filterActions && filterActions.length ? (
```

Comment:
`filterActions` обязательный пропс, может сразу так?
```suggestion
	return filterActions.length ? (
```